src/components/WhyAQE/WhyAQE.tsx

```diff
@@ -0,0 +1,51 @@
+import './WhyAQE.scss';
+import briefcaseIcon from '../../assets/icons/material-symbols_work-outline.svg';
+import batteryIcon from '../../assets/icons/mingcute_battery-4-line.svg';
+import chartIcon from '../../assets/icons/mingcute_chart-bar-2-line.svg';
+import lightbulbIcon from '../../assets/icons/mingcute_light-line.svg';
+
+const WhyAQE = () => {
+  return (
+    <div className="why">
+      <p className="why__title">Why AQE?</p>
+      <div className="why__section">
+        <img src={lightbulbIcon} alt="" className="why__img" />
+        <p className="why__subtitle">Inspiring Environment</p>
+        <p className="why__text">
+          We create stimulating and inspiring atmosphere that encourages
+          creativity, innovation, and a lifelong love for learning among
+          educators.
+        </p>
+      </div>
+      <div className="why__section">
+        <img src={batteryIcon} alt="" className="why__img" />
+        <p className="why__subtitle">Refreshing Experience</p>
+        <p className="why__text">
+          We offer a break from the traditional setting, allowing teachers to
+          rejuvenate their passion for teaching through engaging and interactive
+          camp activities.
+        </p>
+      </div>
```

Comment:
I would be be better to create component for `why_section` insted of repeating code